### DISCUSSION TITLE
“Reply to thread” button text

### DISCUSSION THREAD

**Discussion by hughlilly** [community/community#6178] [Mobile] [answered]
When viewing a Thread, there is a “Reply to thread” button. The words “to thread” are redundant. I’d suggest the button be one of the following:
- Just an icon (fairly standard across other apps, because the icon itself carries enough meaning)
- Icon + “Reply"
- Icon + “Add reply"

Thanks
Hugh

**Comment by taki-on** [comment, marked as answer]
Thank you for the suggestion! We will discuss this in the team!
